apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -975,6 +1004,25 @@ pub(crate) fn metric_query_planning_plan_duration(planner: &'static str, start:
     );
 }
 
+pub(crate) fn metric_rust_qp_init(init_error_kind: Option<&'static str>) {
+    if let Some(init_error_kind) = init_error_kind {
+        u64_counter!(
+            "apollo.router.lifecycle.query_planner.init",
+            "Rust query planner initialisation",
```

Comment:
Nit bc we generally use american
```suggestion
            "Rust query planner initialization",
```

Reply:
ooops british spell checker was on